tests/engines/registries/acceptance/branded/new-test.ts

```diff
@@ -0,0 +1,77 @@
+import { click, currentRouteName } from '@ember/test-helpers';
```

Comment:
Maybe add a test to make sure logged-out users cannot get to this page.

Reply:
Seems less than trivial to test the reroute behavior. `@requireAuth()` will move the entire window to the log-in page and will interrupt tests.

Reply:
You can use `CurrentUserStub` from tests/helpers/require-auth.ts to test this.

Reply:
Tried using `CurrentUserStub` and it does not seem to behave properly for whatever reason in this test... May not be practical to add this test at this time :(